fix-and-improve-appstream-file.patch

```diff
@@ -0,0 +1,99 @@
++    <mediatype>text/x-microdvd</mediatype>
++    <mediatype>text/x-ssa</mediatype>
++  </provides>
++  <launchable type="desktop-id">org.aegisub.Aegisub.desktop</launchable>
++  <translation type="gettext">aegisub</translation>
+   <_name>Aegisub</_name>
+-  <_summary>A free, cross-platform open source tool for creating and modifying subtitles</_summary>
++  <_summary>Create and modify subtitles</_summary>
+   <description>
+     <_p>Aegisub is a free, cross-platform open source tool for creating and modifying subtitles. Aegisub makes it quick and easy to time subtitles to audio, and features many powerful tools for styling them, including a built-in real-time video preview.</_p>
+     <_p>Aegisub was originally created as a tool to make typesetting, particularly in anime fansubs, a less painful experience. At the time of the start of the project, many other programs that supported the Advanced Substation Alpha format lacked (and in many cases, still lack; development on several competing programs have since been dropped for various reasons completely unrelated to Aegisub) many vital functions, or were too buggy and/or unreliable to be really useful.</_p>
+@@ -19,40 +29,43 @@
+       <_li>Fully scriptable through the Automation module</_li>
+     </ul>
+   </description>
+-  <launchable type="desktop-id">aegisub.desktop</launchable>
+-  <kudos>
+-    <kudo>HiDpiIcon</kudo>
+-    <kudo>HighContrast</kudo>
+-    <kudo>UserDocs</kudo>
+-  </kudos>
++  <categories>
++    <category>AudioVideo</category>
++    <category>Graphics</category>
++  </categories>
++  <keywords>
++    <keyword>ass</keyword>
++    <keyword>sub</keyword>
++    <keyword>srt</keyword>
++    <keyword>anime</keyword>
++    <keyword>movie</keyword>
++    <keyword>subtitle</keyword>
++    <keyword>translate</keyword>
++  </keywords>
++  <content_rating type="oars-1.1" />
++  <url type="homepage">http://www.aegisub.org</url>
++  <url type="help">http://docs.aegisub.org</url>
++  <url type="bugtracker">https://github.com/Aegisub/Aegisub/issues</url>
++  <url type="vcs-browser">https://github.com/Aegisub/Aegisub</url>
++  <url type="faq">https://aegisub.org/docs/latest/faq/</url>
++  <developer_name>Aegisub Group</developer_name>
++  <branding>
++    <color type="primary" scheme_preference="light">#EA1D24</color>
++    <color type="primary" scheme_preference="dark">#EA1D24</color>
++  </branding>
+   <screenshots>
+     <screenshot type="default">
+-      <_caption>Typesetting</_caption>
+-      <image>http://static.aegisub.org/img/screenshots/unix/typesetting-efc51b7a.png</image>
+-    </screenshot>
+-    <screenshot>
+-      <_caption>Audio video</_caption>
+-      <image>http://static.aegisub.org/img/screenshots/unix/audio-video-f1f81fc2.png</image>
++      <caption>Audio video</caption>
++      <image xml:lang="en" type="source" width="1600" height="900">https://github.com/Aegisub/Aegisub/assets/65136727/386667e4-df31-476b-8aeb-34bf1fd70bf2</image>
```

Comment:
let's try withou the lang attribute. also `height` and `width` are more problems than they bring.

```suggestion
+      <image type="source">https://github.com/Aegisub/Aegisub/assets/65136727/386667e4-df31-476b-8aeb-34bf1fd70bf2</image>
```